.gitignore

```diff
@@ -1,2 +1,5 @@
 whenchanged
 run.sh
+echo/Echo.class
+echo/EchoTest.class
+.DS_Store
```

Comment:
I append the Java version of `.gitignore` for your project: https://github.com/github/gitignore/blob/master/Java.gitignore. Specifically, `class` files should not be checked into your repo.